media-sound/bitwig-studio/bitwig-studio-4.3.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
The most recent EAPI is EAPI=8. Please consider bumping the ebuild to the most recent EAPI.